test/evp_test.c

```diff
@@ -3569,3 +3578,171 @@ int setup_tests(void)
     ADD_ALL_TESTS(run_file_tests, n);
     return 1;
 }
+
+void cleanup_tests(void)
+{
+    OSSL_PROVIDER_unload(prov_null);
+    OPENSSL_CTX_free(libctx);
+}
+
+/* Done as macros to avoid compiler warnings about unused functions  */
```

Comment:
I don't understand this comment

Reply:
I will delete the comment since @richsalz didnt understand it either :)